e2e/scenario_test.go

```diff
@@ -14,6 +14,52 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v6"
 )
 
+func bootDiagnosticsVMConfigMutator(vmss *armcompute.VirtualMachineScaleSet) {
```

Comment:
I think we should have a separate file/package to house common Scenario config mutators like this

Reply:
honestly we probably want boot diagnostics enabled on all our E2E VMSS models, might be worth adding this as a common mutation